hack/make/check.make

```diff
@@ -1,7 +1,7 @@
 ##@ Check
 ## Targets to be used to ensure code quality
 .PHONY: verify-code
-verify-code: vet lint ## Verify code formatting
+verify-code: vet ## Verify code formatting
```

Comment:
We should probably make this lint target work without having to have the binary pre-installed, but we also don;t need to run the linter as part of this target on CI anyway since we already have a test for that.